packages/AlgernonTests.package/ALGTest.class/instance/resultsClassIncludeName..st

```diff
@@ -0,0 +1,6 @@
+helpers
+resultsClassIncludeName: aNameString 
+	^ self algerInstance itemList items
```

Comment:
Use the helper `self items`

Reply:
I didn't use it, because I tried to improve the performance of the FuzzyMatching tests. Therefore I did not want to put an additional collect (in form of the helper) in front of it.

Reply:
Fair enough.